apparmor.d/groups/lxqt/lxqt-config-brightness

```diff
@@ -22,11 +23,39 @@ profile lxqt-config-brightness @{exec_path} {
   owner /tmp/@{int} r,
 
   @{sys}/class/backlight/ r,
-  @{sys}/devices/@{pci}/**/**/drm/card@{int}/card@{int}-eDP-@{int}/amdgpu_bl@{int}/* rw,
-  @{sys}/devices/@{pci}/**/drm/card@{int}/card@{int}-eDP-@{int}/intel_backlight/* rw,
+  @{sys}/devices/@{pci}/*_backlight/{,max_,actual_}brightness rw,
+  @{sys}/devices/@{pci}/*_backlight/{uevent,type,enabled} r,  
+  @{sys}/devices/@{pci}/backlight/**/{,max_,actual_}brightness rw,
+  @{sys}/devices/@{pci}/backlight/**/{uevent,type,enabled} r,
+  @{sys}/devices/@{pci}/backlight/**/brightness rw,
 
   /dev/tty rw,
 
+  profile pkexec {
+    include <abstractions/base>
+    include <abstractions/app/pkexec>
+    include <abstractions/python>
+
+    ptrace read peer=aa-notify,
+
+    @{sbin}/apparmor_parser Px,
+    @{lib}/@{python_name}/site-packages/apparmor/update_profile.py ix,
+
+    /usr/share/apparmor/** r,
+    /usr/share/terminfo/** r,
+
+    @{etc_ro}/inputrc r,
+    @{etc_ro}/inputrc.keys r,
+
+    /etc/apparmor.d/ r,
+    /etc/apparmor.d/** rw,
+    /etc/apparmor/* r,
+
+    @{PROC}/@{pid}/mounts r,
+
+    include if exists <local/lxqt-config-brightness_pkexec>
+  }
```

Comment:
No, I did not ask you to fully copy paste the pkexec subprofile from aa-notify.

By design, this subprofile only allow editing and reloading an apparmor profile. Here you need to only allow the things `lxqt-config-brightness` needs to do with privileged right.